application/controllers/collection_instrument.py

```diff
@@ -85,20 +86,29 @@ def upload_instrument(self, exercise_id, file, ru_ref=None, classifiers=None, se
         :return: a collection instrument instance
         """
 
+        try:
+            seft_ci_bucket = GoogleCloudSEFTCIBucket(current_app.config)
+            path = seft_ci_bucket.upload_file_to_bucket(file=file)
+        except Exception as e:
+            log.error("An error occurred when trying to put SEFT CI in bucket")
+            log.error(e)
```

Comment:
You could .exception here as that automatically logs the error stack trace. e.g.
```
log.exception("An error occurred when trying to put SEFT CI in bucket")
```